frontend/admin/src/js/components/pool/CreatePool.tsx

```diff
@@ -2,102 +2,81 @@ import * as React from "react";
 import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
 import { useIntl } from "react-intl";
 import { toast } from "react-toastify";
-import {
-  Input,
-  MultiSelect,
-  Select,
-  Submit,
-  TextArea,
-} from "@common/components/form";
-import { getLocale } from "@common/helpers/localize";
-import { notEmpty } from "@common/helpers/util";
+import { Input, Select, Submit } from "@common/components/form";
+import { unpackMaybes } from "@common/helpers/formUtils";
 import { navigate } from "@common/helpers/router";
 import { errorMessages } from "@common/messages";
-import { keyStringRegex } from "@common/constants/regularExpressions";
-import { enumToOptions } from "@common/helpers/formUtils";
-import {
-  getOperationalRequirement,
-  getPoolStatus,
-  OperationalRequirementV2,
-} from "@common/constants/localizedConstants";
+import { getGenericJobTitlesWithClassification } from "@common/constants/localizedConstants";
 import Pending from "@common/components/Pending";
+import PageHeader from "@common/components/PageHeader/PageHeader";
+import Breadcrumbs, {
+  BreadcrumbsProps,
+} from "@common/components/Breadcrumbs/Breadcrumbs";
+import { ViewGridIcon } from "@heroicons/react/outline";
+import { ViewGridIcon as SolidGridIcon } from "@heroicons/react/solid";
+import Link from "@common/components/Link/Link";
 import { useAdminRoutes } from "../../adminRoutes";
 import {
-  Classification,
-  CmoAsset,
-  CreatePoolInput,
-  CreatePoolMutation,
-  Pool,
-  useCreatePoolMutation,
-  useGetCreatePoolDataQuery,
-  User,
-  PoolStatus,
+  CreatePoolAdvertisementInput,
+  useCreatePoolAdvertisementMutation,
+  CreatePoolAdvertisementMutation,
+  useGetMePoolCreationQuery,
+  GenericJobTitleKey,
 } from "../../api/generated";
 import DashboardContentContainer from "../DashboardContentContainer";
 
 type Option<V> = { value: V; label: string };
 
-type FormValues = Pick<
-  Pool,
-  "description" | "operationalRequirements" | "keyTasks" | "status"
-> & {
+type FormValues = {
   key: string;
-  name: {
-    en: string;
-    fr: string;
-  };
-  assetCriteria: string[] | undefined;
-  classifications: string[] | undefined;
-  essentialCriteria: string[] | undefined;
-  owner: string;
+  classification: string[] | undefined;
```

Comment:
Would it be better to just mark this as optional?

```suggestion
  classification?: string[];
```

Reply:
given it is required in the form, I dropped the undefined portion of it, hopefully that isn't worse